File: python/project_013/main.py
num = int(input("Enter a number: "))
if num < 2:
    print(f"{num} is not prime.")
else:
    for i in range(2, int(num ** 0.5) + 1):
        if num % i == 0:
            print(f"{num} is not prime.")
            break
    else:
        print(f"{num} is prime.")

File: rust/project_013/main.rs
use std::io;

fn main() {
    println!("Enter numbers separated by spaces:");
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read");
    let numbers: Vec<i32> = input.trim().split_whitespace().filter_map(|s| s.parse().ok()).collect();
    match numbers.iter().max() {
        Some(max) => println!("Maximum: {}", max),
        None => println!("No numbers entered."),
    }
}
